src/common.ts

```diff
@@ -17,19 +17,24 @@ import {URL} from 'url';
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
 export class GaxiosError<T = any> extends Error {
-  code?: string;
+  code?: Number;
```

Comment:
I think this should be a `string` to match this convention:
https://nodejs.org/api/errors.html#errorcode